ProcessesApi/V1/Services/SoleToJointService.cs

```diff
@@ -110,24 +110,16 @@ private async Task CheckTenureInvestigation(StateMachine<string, string>.Transit
             SoleToJointHelpers.ValidateFormData(formData, expectedFormDataKeys);
             var tenureInvestigationRecommendation = formData[SoleToJointFormDataKeys.TenureInvestigationRecommendation].ToString();
 
-            switch (tenureInvestigationRecommendation)
+            var mapping = new Dictionary<string, string>
```

Comment:
May be clearer if rather than `mapping` it's called triggerMappings or something? Not sure, but something more clear for other devs